pkg/cluster/internal/loadbalancer/const.go

```diff
@@ -20,7 +20,7 @@ package loadbalancer
 const ControlPlanePort = 6443
 
 // Image defines the loadbalancer image:tag
-const Image = "nginx:1.15.12-alpine"
+const Image = "kindest/haproxy:2.0.0-alpine"
```

Comment:
I remember talking about prepulling the `loadbalancer` image before but don´t remember the conclusion 😅 , should we prepull it?

Reply:
It's not prepulled because it runs on the host directly as a node. It only needs to pull once though.